srcpkgs/thinkpad-scripts/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'thinkpad-scripts'
+pkgname=thinkpad-scripts
+version=4.11.0
+revision=1
+build_style=gnu-makefile
```

Comment:
shouldn't this be a python3-module then ?

Reply:
Honestly, I didn't look closely enough to have any idea how the internals of this package work. I saw that it had a Makefile and just started with that and then later saw the installation instructions wanted you to do something with setup.py. I'll see if I can package it as `build_style=python3-module` tomorrow.